src/plugins/expression_shape/public/components/shape/shape_component.tsx

```diff
@@ -78,16 +75,14 @@ function ShapeComponent({
 
   return (
     <div className="shapeAligner">
-      <ShapeDrawer
-        shapeType={shapeType}
-        shapeContentAttributes={shapeContentAttributes}
-        shapeAttributes={shapeAttributes}
-        ref={shapeRef}
-      />
+      <EuiErrorBoundary>
```

Comment:
error boundary not needed. Only use error boundary at top level where async tree is started